tests/test_server.py

```diff
@@ -2824,6 +2825,100 @@ def test_check_backup_validity_under_minimum_size(self, server, capsys):
             in out
         )
 
+    @patch("barman.infofile.LocalBackupInfo.walk_to_root")
+    @patch("barman.server.Server.get_children_timelines")
+    @patch("barman.server.Server.get_wal_info")
+    @patch("barman.backup.BackupManager.get_next_backup")
+    @patch("barman.backup.BackupManager.get_previous_backup")
+    def test_get_backup_ext_info(
+        self,
+        prev_backup_mock,
+        next_backup_mock,
+        wal_info_mock,
+        children_timeline_mock,
+        walk_to_root_mock,
+    ):
+        """
+        TODO
```

Comment:
Leftover?

Reply:
yes, i ll remove that

Reply:
I think the docstring is weird as well, right?

Reply:
It is just a dict not json

Reply:
Indeed!